### PR TITLE
Multi-look Averaging Method

### PR DESCRIPTION
**Pull request by hb6688**
Added multi-look averaging method to complement the existing resampling method.

Updated dswx_sar_util.py as well.

dowsample_20_to_30

### CODE REVIEW

**Review by oberonia78 — APPROVED** [review]
LGTM. Once @hb6688 push the new changes, please merge the PR. Thanks @hb6688

**Comment by hb6688** [comment]
I am going to merge this PR after latest updates.  Please test run it Jungkyo.

**Pull request merged**
